tests/unit/api/test_delete_importable_dataset_api.py

```diff
@@ -0,0 +1,16 @@
+from flask import url_for
+import os
+
+def test_delete_importable_datasets_api(flask_app):
+    f = open("tests/resources/input_directory/DATASET_THAT_SHOULD_BE_DELETED", "x")
```

Comment:
the files themselves are still named .tar 👍🏻 

but the file extension should not be part of the API so:
- the api is called with /DATASET_THAT_SHOULD_BE_DELETED
- validate the dataset name
- local storage deletes the file DATASET_THAT_SHOULD_BE_DELETED.tar